src/node_crypto.cc

```diff
@@ -2921,6 +2923,7 @@ void CipherBase::SetAuthTag(const FunctionCallbackInfo<Value>& args) {
   }
 
   cipher->auth_tag_len_ = tag_len;
+  cipher->auth_tag_state_ = kAuthTagKnown;
```

Comment:
Is there a chance that this line or one of the above will be run after `MaybePassAuthTagToOpenSSL()` is, i.e. could this reset the state from `kAuthTagPassedToOpenSSL` to a lower value?

Reply:
Yes. I was planning to throw in that case, but that would be semver-major...